black.py

```diff
@@ -785,13 +783,13 @@ def show(cls, code: Union[str, Leaf, Node]) -> None:
 STARS = {token.STAR, token.DOUBLESTAR}
 VARARGS_PARENTS = {
     syms.arglist,
-    syms.argument,  # double star in arglist
-    syms.trailer,  # single argument to call
+    syms.argument,  # Double star in arglist.
+    syms.trailer,  # Single argument to call.
     syms.typedargslist,
-    syms.varargslist,  # lambdas
+    syms.varargslist,  # Lambdas.
```

Comment:
But those above? Those aren't sentences, they aren't meant to be sentences. They are semantic explanations of the node codes in this context.